template/project.yml

```diff
@@ -58,6 +58,11 @@ target-hooks:
     dir: ${docroot}/profiles/contrib/lightning
     command: npm run install-libraries
 
+# Configuration management options, such as core CM or features.
+# cm:
```

Comment:
I thought about that, but it won't work for a lot of existing projects that already have different machine and bundle names. It also wouldn't let you specifically disable features integration.

Note that if you provide a non-existent bundle name, the features revert will still run but will throw a warning.

If you don't want to introduce a new top-level config, we could just put this under "project" for now until we decide to expand CM support.